app/controllers/v0/sessions_controller.rb

```diff
@@ -89,6 +99,7 @@ def saml_callback
         obscure_token = Session.obscure_token(@session.token)
         Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
         Benchmark::Timer.stop(TIMER_LOGIN_KEY, @saml_response.in_response_to, tags: ['status:success'])
+        StatsD.increment("api.auth.login_callback.#{context_key}.success")
```

Comment:
What if we add this as a tag to the timer?

`tags: ['status:success', "context:#{context_key}]"`

Reply:
@patrickvinograd would this end up being cleaner maybe? Not sure how tags work in StatsD, but I know in Sentry they're actually a bit more searchable and since this is based on context, that might prove useful should we need to track the context through controller actions.

Reply:
@mloberg How do tags flow through to prometheus? I'd like to be able to differentiate login counts/rates for each of the different callbacks there.

Reply:
You need to have _key:value_ in StatsD, that should flow to Prometheus correctly then.

e.g. `api.auth.login[status:success]` in StatsD will be `api_auth_login{status=success}` in Prometheus.